app/js/map/map.factory.js

```diff
@@ -58,6 +59,12 @@
               return map.getBounds();
             }
 
+            function createLatLng(latitude, longitude) {
```

Comment:
Also 'myLatLng' doesn't make that much sense in this factory function kinda context, change it to just latLng or coordinate or something